### PR TITLE
fix(css): ignore `media` params with scss interpolation

### LINKED ISSUE #3719: [scss] bug with interpolation inside media
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEABAtnAJgSwIYAEAzmAE5wIF5RYEDEwAJEegLTQA2AngDQGMcA5q2pZWAVwAOAXwLAAOlALKCkDhFJJ6ABl0BuRdMUgeICJJg5oRZKDylSEAO4AFewhso8ANwg4sJiB4RDDIAGZ4HERwpgBGpHhgANZwMADKkok4UILIMKTiMSBYEGDhkdGm2dGkMC4Jguh45VFFAFZEAB4AQgnJqWl4mAAy2XAtlWbiMJLTAEwTRZmkNcggJEQ2ppKk2TAA6v4wABbIABza247R+wmSaztwNd7jpuQAjuI45PV4jc1ICKtUzRdA4PIFIpEbKCDhwACK4gg8EWphgeFihywJ2QczRCRwHBhAGEIOgmmsoNBXiBxNEACoYzz5QrSaRAA)
```sh
--parser scss
```

**Input:**
```scss
$sm-only: '(min-width: 768px) and (max-width: 991px)';
$lg-and-up: '(min-width: 1200px)';

@media screen and #{$sm-only, $lg-and-up} {
    color: #000;
}

```

**Output:**
```jsx
@media screen and,  {
	color: #000;
}

```

**Expected behavior:**

```scss
$sm-only: '(min-width: 768px) and (max-width: 991px)';
$lg-and-up: '(min-width: 1200px)';

@media screen and #{$sm-only, $lg-and-up} {
    color: #000;
}

```

Seems problem inside `postcss-media-query-parser`

### PR DESCRIPTION
**Pull request by evilebottnawi**
Fixes #3719 

Right now media parser is very buggy for scss, i think better print media params with interpolation as is (it is don't break code).

### CODE REVIEW

File: src/language-css/parser-postcss.js [hunk 1]
```diff
@@ -211,6 +211,14 @@ function parseNestedCSS(node) {
           value: node.params
         };
       } else {
+        if (node.params.indexOf("#{") !== -1) {
```
Comment (lydell) [inline src/language-css/parser-postcss.js]: `node.params.includes("#{")`?

Comment (j-f1) [reply src/language-css/parser-postcss.js]: That’ll [have to wait](http://node.green/#ES2016-features-Array-prototype-includes) until April (or until we drop support for Node 4), unfortunately.

Comment (lydell) [reply src/language-css/parser-postcss.js]: I think you confuse `String.prototype.includes` and `Array.prototype.includes`. The string one is available in Node 4.

**Comment by evilebottnawi** [comment]
@lydell done, maybe new eslint rule will allow to catch this stuff

**Pull request merged**